### PR TITLE
redo DEV-2037

### PR DESCRIPTION
**Pull request by x110dc**
See https://github.com/newsrevenuehub/rev-engine/pull/530

### CODE REVIEW

**Review by njharman — APPROVED** [review]
Don't forget to log your time on ticket.

test coverage ok

File: pyproject.toml [hunk 1]
```diff
@@ -71,6 +71,7 @@ whitenoise = "^6.2.0"
 django-reversion-compare = "^0.15.0"
 django-migration-linter = "^4.1.0"
 django-test-migrations = "^1.2.0"
+Faker = "^13.15.0"
```
Comment (x110dc) [inline pyproject.toml:74]: Should this be in `dev-dependencies`?

File: apps/users/constants.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Used in requests to bad actor api that don't involve actual donations
+BAD_ACTOR_FAKE_AMOUNT = 1000
```
[the remark this answers is not in the document]
Comment (benjaminEwhite) [reply apps/users/constants.py:2]: Acting on both comments.

File: pyproject.toml (hunk 1 above)
Comment (benjaminEwhite) [reply pyproject.toml]: yes. moving.

**Pull request merged**